calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,141 @@
+| `radar_initial_crop_box_min_x`              | `double`      | `-50.0`                                                 | Minimum x-coordinate for the initial radar calibration area.                                                                                      |
+| `radar_initial_crop_box_min_y`              | `double`      | `-50.0`                                                 | Minimum y-coordinate for the initial radar calibration area.                                                                                      |
+| `radar_initial_crop_box_min_z`              | `double`      | `-50.0`                                                 | Minimum z-coordinate for the initial radar calibration area.                                                                                      |
+| `radar_initial_crop_box_max_x`              | `double`      | `50.0`                                                  | Maximum x-coordinate for the initial radar calibration area.                                                                                      |
+| `radar_initial_crop_box_max_y`              | `double`      | `50.0`                                                  | Maximum y-coordinate for the initial radar calibration area.                                                                                      |
+| `radar_initial_crop_box_max_z`              | `double`      | `50.0`                                                  | Maximum z-coordinate for the initial radar calibration area.                                                                                      |
+| `lidar_background_model_leaf_size`          | `double`      | `0.1`                                                   | Voxel size in meter for the lidar background model.                                                                                               |
+| `radar_background_model_leaf_size`          | `double`      | `0.1`                                                   | Voxel size in meter for the radar background model.                                                                                               |
+| `max_calibration_range`                     | `double`      | `50.0`                                                  | Maximum range for calibration in meters.                                                                                                          |
+| `background_model_timeout`                  | `double`      | `5.0`                                                   | The background model will terminate if it is not updated within this period, measured in seconds.                                                 |
+| `min_foreground_distance`                   | `double`      | `0.4`                                                   | Square of this value in meters is used for filtering foreground points, typically needing to be at least double the `background_model_leaf_size`. |
```

Comment:
The mention of using the squared distance makes it only more confusing.
The only reason the squared value is used in the code is because how K-nearest search returns the values.
Please describe the parameters by their semantic meaning rather than how they are used in the code. If you do not understand at a simple glance, there is no avoiding studying the code

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/5702a11535c36457480e3607e01a615e82c88bc1